www/js/survey/enketo/enketo-time-use.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Directive to display a survey for each trip
+ * Assumptions:
+ * - The directive is embedded within an ion-view
+ * - The controller for the ion-view has a function called
+ *      'recomputeDisplayTrips` which modifies the trip *list* as necessary. An
+ *      example with the label view is removing the labeled trips from the
+ *      "toLabel" filter. Function can be a no-op (for example, in
+ *      the diary view)
+ * - The view is associated with a state which we can record in the client stats.
+ * - The directive implements a `verifyTrip` function that can be invoked by
+ *      other components.
+ */
+
+angular.module('emission.survey.enketo.time-use',
+    ['emission.stats.clientstats',
+        'emission.services',
+        'emission.survey.enketo.launch',
+        'emission.enketo-survey.answer',
+        'emission.survey.enketo.preview',
+        'emission.survey.inputmatcher'])
+.directive('enketoTimeuseButton', function() {
+  return {
+    scope: {
+    },
+    controller: "EnketoTimeuseButtonCtrl",
+    templateUrl: 'templates/survey/enketo/timeuse-button.html'
+  };
+})
+.directive('enketoTimeuseInline', function() {
```

Comment:
I removed the `timeuse-inline` related code in my latest commit - since we are not using the inline implementation for timeuse, there is no reason to have it.